### PR TITLE
feat(pingcap/tidb) report build url info for flaky and case time report records

### LINKED ISSUE #2370: report build url info for flaky and case time report records

### PR DESCRIPTION
**Pull request by wuhuizuo**
Fixes #2370

We need the build url to deep dialog the failure or slow problem of test cases.

Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### CODE REVIEW

File: pipelines/pingcap/tidb/latest/ghpr_unit_test.groovy [hunk 1]
```diff
@@ -75,9 +75,10 @@ pipeline {
                     sh label: "Parse flaky test case results", script: './scripts/plugins/analyze-go-test-from-bazel-output.sh tidb/bazel-test.log || true'
                     container('deno') {
                         sh label: "Report flaky test case results", script: """
-                            deno run --allow-all http://fileserver.pingcap.net/download/ci/scripts/plugins/report-flaky-cases.ts \
+                            deno run --allow-all http://fileserver.pingcap.net/download/ci/scripts/plugins/report-flaky-cases-v20230821.ts \
```
Comment (purelind) [inline pipelines/pingcap/tidb/latest/ghpr_unit_test.groovy:78]: It seem this file `report-flaky-cases-v20230821.ts` not exist now.

Comment (wuhuizuo) [reply pipelines/pingcap/tidb/latest/ghpr_unit_test.groovy:78]: no, I uploaded it.

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**